generators/server/templates/src/main/java/package/gateway/_TokenRelayFilter.java

```diff
@@ -0,0 +1,37 @@
+package <%=packageName%>.gateway;
+
+import com.netflix.zuul.ZuulFilter;
+import com.netflix.zuul.context.RequestContext;
+import org.springframework.stereotype.Component;
+
+import java.util.Set;
+
+@Component
+public class TokenRelayFilter extends ZuulFilter {
+    @Override
+    public Object run() {
+        RequestContext ctx = RequestContext.getCurrentContext();
+
+        // Alter ignored headers as per: https://gitter.im/spring-cloud/spring-cloud?at=56fea31f11ea211749c3ed22
```

Comment:
I'll change the comment as gitter is not persistent (messages are lost after 15 days)